.github/workflows/tox.yml

```
@@ -27,21 +27,21 @@ jobs:
                 }}
           is-not-python38:
             - ${{
-                  (github.repository == 'stackhpc/kayobe') &&
-                  ((github.base_ref == 'stackhpc/2024.1') ||
-                  (github.ref == 'refs/heads/stackhpc/2024.1') ||
+                  ((github.repository == 'stackhpc/kayobe') &&
+                  (github.base_ref == 'stackhpc/2024.1')) ||
+                  ((github.repository == 'stackhpc/kayobe') &&
+                  (github.ref == 'refs/heads/stackhpc/2024.1')) ||
```

This might be clearer?
```suggestion
                  ((github.repository == 'stackhpc/kayobe') &&
                  ((github.base_ref == 'stackhpc/2024.1') ||
                  (github.ref == 'refs/heads/stackhpc/2024.1'))) ||
```